### ISSUE TITLE
3.1.7 Plain Language (Minimum) - Too much overlap with existing criteria

### ISSUE THREAD

**Issue by FionaHolder** [w3c/wcag21#179]
It does seem that this would be better addressed as a revision to 3.3.2 Labels or Instructions and/or 2.4.6 Headings and Labels, though I understand that WCAG 2.1 are not revising any existing criteria. However, I feel that there is a huge amount of overlap between this and the other similar criteria. It is much better for developers, testers and automated testing tools if the criteria are very defined so that it is clear where a failure lies. 
As proposed, if I were to find instruction text unclear, that would fail both 3.3.2 and this new criterion. I feel that it is confusing for developers and testers when this occurs, and adds to the overwhelm felt when a huge number of new criteria are introduced. 

If the existing criteria cannot be revised with 2.1, I'd advise waiting until the next WCAG revision where they could be revised, rather than causing confusion with a new one. 

http://myaccessible.website/2017/03/wcag-2-1-review-3-1-7-plain-language-minimum/

**Comment by lseeman** [comment]
refrencing this comment  #30

**Comment by awkawk** [comment]
Thanks for the comment - the Working Group has not reached consensus on this proposed SC so it is deferred for future consideration.
